### PR TITLE
Return valid json from StatusHandler

### PR DESCRIPTION
**Pull request by mattrobenolt**

### CODE REVIEW

**Comment by RichardScothern** [comment]
LGTM

File: health/health.go [hunk 1]
```diff
@@ -199,7 +199,7 @@ func StatusHandler(w http.ResponseWriter, r *http.Request) {
 
 		// Parsing of the JSON failed. Returning generic error message
 		if err != nil {
-			w.Write([]byte("{server_error: 'Could not parse error message'}"))
+			w.Write([]byte(`{"server_error": "Could not parse error message"}`))
```
Comment (stevvooe) [inline health/health.go]: Ack! How did this slip through!

Let's take this further:

```go
json.NewEncoder(w).Encode(struct { 
    ServerError string `json:"server_error"` 
}{
    ServerError: "Could not parse error message"})
```

**Comment by stevvooe** [comment]
@mattrobenolt Could you please update based on feedback?

**Comment by mattrobenolt** [comment]
@stevvooe feedback addressed

**Comment by stevvooe** [comment]
@mattrobenolt Thanks!

LGTM.

**Pull request merged**